scripts/utils/release-utils.ts

```diff
@@ -90,14 +90,34 @@ export function prettyVersionDiff(oldVersion: string, inc: any): string {
 }
 
 /**
- * Write CHANGELOG.md to disk. Stencil uses the Angular-variant of conventional commits; commits must be formatted
- * accordingly in order to be added to the changelog properly.
+ * Write changes to the local CHANGELOG.md on disk.
+ *
+ * Stencil uses the Angular-variant of conventional commits; commits must be formatted accordingly in order to be added
+ * to the changelog properly.
  * @param opts build options to be used to update the changelog
  */
 export async function updateChangeLog(opts: BuildOptions): Promise<void> {
   const ccPath = join(opts.nodeModulesDir, '.bin', 'conventional-changelog');
+  const ccConfigPath = join(opts.scriptsBuildDir, 'utils', 'conventional-changelog-config.js');
   const { execa } = await import('execa');
-  await execa('node', [ccPath, '-p', 'angular', '-o', '-i', opts.changelogPath, '-s'], { cwd: opts.rootDir });
+  // API Docs for conventional-changelog: https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-core#api
+  await execa(
+    'node',
+    [
+      ccPath,
+      '--preset',
```

Comment:
I expanded the short flags into their long forms. Some of these single letter forms are kinda confusing on their own!

Reply:
Yes, agree. Much more readable